### PR TITLE
Backlog 14140 integrate tree facet in ui

### PR DESCRIPTION
**Pull request by kovunov**
## JIRA

https://jira.jahia.org/browse/BACKLOG-14140

## Description
Implemented ui for hierarchical facets

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "@elastic/react-search-ui-views": "1.4.1",
     "@elastic/search-ui": "1.4.1",
     "@jahia/i18next": "^1.0.0-beta.20180628145923",
-    "@jahia/search-ui-jahia-connector": "^1.0.0",
+    "@jahia/search-ui-jahia-connector": "../search-ui-jahia-connector",
```
Comment (AKarmanov) [inline package.json]: We need to put todo somewhere about this one so that we know why it doesn't build ;)

File: src/javascript/app/App.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import '@elastic/react-search-ui-views/lib/styles/styles.css';
-import JahiaSearchAPIConnector, {Field, FieldType} from '@jahia/search-ui-jahia-connector';
+import JahiaSearchAPIConnector, {Field, FieldType} from '../../../../search-ui-jahia-connector/src';
```
Comment (AKarmanov) [inline src/javascript/app/App.jsx]: You could use yalk to keep the import name the way it is, only package.json import would change. As it stands right now nobody would be able to run the module unless there is the same file structure.

File: package.json (hunk 1 above)
Comment (kovunov) [reply package.json]: I think this one is outdated, I changed to reference the one in npm registry.

File: src/javascript/app/App.jsx (hunk 2 above)
Comment (kovunov) [reply src/javascript/app/App.jsx]: Same, this was one of the first commits, I changed it to be good I guess)

**Review by AKarmanov — APPROVED** [review]

**Pull request merged**